scripts/FCCee/ALLEGRO/ALLEGRO_o1_v03/ALLEGRO_make_TH1.py

```diff
@@ -0,0 +1,154 @@
+                  
+
+
+
+  #############################################################################
+  ##                                                                         ## 
+  ##                BEGIN: ECal Barrel Histogram Definition                  ##
+  ##                                                                         ## 
+  #############################################################################
+
+  dir_ECalBarrel = outputFile.mkdir("ECalBarrel")
+
+  hist_ccE = ROOT.TH1F("h_CaloCluster_E",
+                       "CaloCluster Energy;Energy [MeV];Counts / 0.15 MeV",
+                       100, 0, 15)
+  hist_ctcE = ROOT.TH1F("h_CaloTopoCluster_E",
+                        "CaloTopoCluster Energy;Energy [MeV];Counts / 0.15 MeV",
+                        100, 0, 15)
+  hist_ecal_totE = ROOT.TH1F("h_ECalBarrelModuleThetaMergedPosition_totE",
+                             "ECalBarrelModuleThetaMergedPosition total Energy per evt;Energy [MeV];Counts / 0.15 MeV",
+                             100, 0, 15)
+  hist_ecal_posX = ROOT.TH1F("h_ECalBarrelModuleThetaMergedPosition_posX",
+                             "ECalBarrelModuleThetaMergedPosition position X;X [mm];Counts / 37 mm",
+                             150, -2770, 2770)
+  hist_ecal_posY = ROOT.TH1F("h_ECalBarrelModuleThetaMergedPosition_posY",
+                             "ECalBarrelModuleThetaMergedPosition position Y;Y [mm];Counts / 37 mm",
+                             150, -2770, 2770)
+  hist_ecal_posZ = ROOT.TH1F("h_ECalBarrelModuleThetaMergedPosition_posZ",
+                             "ECalBarrelModuleThetaMergedPosition position Z;Z [mm];Counts / 41 mm",
+                             150, -3100, 3100)
+  
+  dir_list.append(dir_ECalBarrel)
+  histo_list.append([hist_ccE, 
+                     hist_ctcE,
+                     hist_ecal_totE,
+                     hist_ecal_posX,
+                     hist_ecal_posY,
+                     hist_ecal_posZ])
+
+  #############################################################################
+  ##                                                                         ##  
+  ##                  END: ECal Barrel Histogram Definition                  ##
+  ##                                                                         ## 
+  #############################################################################
+  
+
+
+
+  # Loop over dataset
+  for event in podio_reader.get("events"):
+
+
+    ###########################################################################
+    ##                                                                       ## 
+    ##                  BEGIN: ECal Barrel Event Loop                        ##
+    ##                                                                       ## 
+    ###########################################################################
+
+    for calo in event.get("CaloClusters"):
+      energy =  calo.getEnergy()
+      hist_ccE.Fill(energy)
```

Comment:
```suggestion
      hist_ccE.Fill(calo.getEnergy())
```
It's easier to read and the intermediate variable isn't doing anything. Same comment for other lines.